bin/util.js

```diff
@@ -0,0 +1,130 @@
+const _ = require('lodash');
+
+module.exports = {
+
+    /**
+     * Extract selected options in the provided command.
+     * Omits commander private variables and other objects.
+     *
+     * @param {Object} command - Commander.Command Instance
+     * @returns {Object} - Extracted options from command
+     */
+    commanderToObject: (command) => {
+        let obj = {
+            command: command.name(),
+            version: command.parent._version || false
+        };
+
+        // Iterate over all the command properties
+        _.forEach(Object.keys(command), (prop) => {
```

Comment:
This can be cleaned up by assigning the result of a `reduce` call to `obj` first, and assigning the `command` and `version` properties.